Parse/Public/ParseObject.cs

```diff
@@ -428,8 +428,8 @@ string propertyName
     /// <param name="traverseParseObjects">Whether to traverse into ParseObjects' children</param>
     /// <param name="yieldRoot">Whether to include the root in the result</param>
     /// <returns></returns>
-    internal static IEnumerable<object> DeepTraversal(
-        object root, bool traverseParseObjects = false, bool yieldRoot = false) {
+    internal static IEnumerable<object> DeepTraversal(object root,
+        bool traverseParseObjects = false, bool yieldRoot = false) {
```

Comment:
Necessary changes? I think we can either put these in another PR or just discard them, no?